src/integrationTest/java/uk/gov/hmcts/reform/demo/config/SwaggerPublisher.java

```diff
@@ -28,8 +27,6 @@
     private MockMvc mvc;
 
     @DisplayName("Generate swagger documentation")
-    @Test
-    @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
why remove this?

Reply:
See my comment above.

Maybe we can delete this class?

Reply:
Or I can fix this test because now codecov report is failing

Reply:
Eventually I have fixed the test.